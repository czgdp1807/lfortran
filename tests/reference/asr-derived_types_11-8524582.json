{
    "basename": "asr-derived_types_11-8524582",
    "cmd": "lfortran --indent --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/derived_types_11.f90",
    "infile_hash": "8898cd8809e0f2bdc067b0df605400ec0d1685ba87e1a1be88bc25ad",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-derived_types_11-8524582.stdout",
    "stdout_hash": "e4e87bb321424764b1c2db37075bdeae175aea18fa78e4bd8e1c62b0",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}