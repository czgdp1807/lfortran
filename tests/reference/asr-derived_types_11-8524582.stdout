(TranslationUnit
    (SymbolTable
        1
        {
            derived_types_10:
                (Module
                    (SymbolTable
                        2
                        {
                            type_y:
                                (Variable
                                    2
                                    type_y
                                    []
                                    Local
                                    ()
                                    ()
                                    Parameter
                                    (Struct
                                        2 y
                                        []
                                    )
                                    Source
                                    Public
                                    Required
                                    .false.
                                ),
                            x:
                                (Variable
                                    2
                                    x
                                    [type_y]
                                    Local
                                    (StringConcat
                                        (StringConstant
                                            "  "
                                            (Character 1 2 () [])
                                        )
                                        (StructInstanceMember
                                            (Var 2 type_y)
                                            3 backslash
                                            (Character 1 1 () [])
                                            ()
                                        )
                                        (Character 1 3 () [])
                                        ()
                                    )
                                    ()
                                    Save
                                    (Character 1 3 () [])
                                    Source
                                    Public
                                    Required
                                    .false.
                                ),
                            y:
                                (StructType
                                    (SymbolTable
                                        3
                                        {
                                            backslash:
                                                (Variable
                                                    3
                                                    backslash
                                                    []
                                                    Local
                                                    (StringConstant
                                                        "\"
                                                        (Character 1 1 () [])
                                                    )
                                                    ()
                                                    Save
                                                    (Character 1 1 () [])
                                                    Source
                                                    Public
                                                    Required
                                                    .false.
                                                )
                                        })
                                    y
                                    []
                                    [backslash]
                                    Source
                                    Public
                                    .false.
                                    ()
                                    ()
                                )
                        })
                    derived_types_10
                    []
                    .false.
                    .false.
                )
        })
    []
)
